src/spectrecoff/CanvasImage.fs

```diff
@@ -0,0 +1,22 @@
+[<AutoOpen>]
+module SpectreCoff.CanvasImage
+
+open System
+open System.IO
+open Spectre.Console
+
+let mutable maxWidth = 16
+
+type ImageSource =
+| Bytes of Byte[]
+| Stream of Stream
+| Path of String
```

Comment:
Not Uri, or does Spectre also just accept a string?

Reply:
As discussed, no URI.